corehq/apps/reports/tasks.py

```diff
@@ -145,14 +142,23 @@ def _send_email(user, report, hash_id, recipient):
     send_report_download_email(report.name, recipient, link)
 
 
-def _store_excel_in_redis(report_class, file):
-    hash_id = uuid.uuid4().hex
+def _store_excel_in_blobdb(report_class, file, domain):
 
-    r = get_redis_client()
-    r.set(hash_id, [report_class, file.getvalue()])
-    r.expire(hash_id, EXPIRE_TIME)
+    key = uuid.uuid4().hex
+    expired = 60 * 24 * 7  # 7 days
+    db = get_blob_db()
 
-    return hash_id
+    kw = {
+        "domain": domain,
+        "parent_id": key,
+        "type_code": CODES.tempfile,
```

Comment:
not sure if the code should be data_export or a new code? Its unclear how important that is to me

Reply:
I can change it to `data_export`, which makes more sense as a variable name, I don't know enough about what's happening here to know whether that will break stuff though.